lms/static/scripts/frontend_apps/index.js

```diff
@@ -1,29 +1,39 @@
-// Polyfills.
 import 'focus-visible';
-
-// Setup app.
 import { createElement, render } from 'preact';
 
 import { readConfig, Config } from './config';
 import BasicLtiLaunchApp from './components/BasicLtiLaunchApp';
 import CanvasOAuth2RedirectErrorApp from './components/CanvasOAuth2RedirectErrorApp';
 import FilePickerApp from './components/FilePickerApp';
 import { ClientRpc } from './services/client-rpc';
+import {
+  APIService,
+  GradingService,
+  VitalSourceService,
+  Services,
+} from './services';
 
 /** @typedef {import('./services/client-rpc').ClientConfig} ClientConfig */
 
-const rootEl = document.querySelector('#app');
-if (!rootEl) {
-  throw new Error('#app container for LMS frontend is missing');
-}
```

Comment:
`rootEl` initialization is moved closer to the bottom of the file where it is used.